src/utils/Utils.ts

```diff
@@ -103,6 +104,25 @@ export class FileUtils {
     public static getExpiresBySize(filesize: number): number {
         return Date.now() + this.getTimeLeftBySize(filesize);
     }
+
+    public static async getFilesCount(): Promise<number> {
+        try {
+            const realFiles = await fs.readdir(filesDir, { withFileTypes: true });
+            const fileCount = (
+                await Promise.all(
+                    realFiles.map(item => {
+                        if (item.isDirectory()) {
+                            return 0;
+                        }
+                        return 1;
+                    }),
+                )
+            ).reduce((acc, cur) => acc + cur, 0);
+            return fileCount;
+        } catch {
+            return 0;
+        }
+    }
```

Comment:
can't this simply be:
```ts
try {
	const realFiles = await fs.readdir(filesDir, { withFileTypes: true });
	return realFiles.length;
} catch {
	return 0;
}
```

Reply:
Yeah, guess there will never be a directory in there

Reply:
> Yeah, guess there will never be a directory in there

yes, but even if we did, you can pass `recursive: true` as a prop into `readdir`, however, if there is a dir in that folder, a lot of things would break